src/js/serial_backend.js

```diff
@@ -362,6 +363,26 @@ function abortConnect() {
     clicks = false;
 }
 
+async function processBuildConfiguration() {
```

Comment:
this should probably be called as part of the autoDetect method?

Reply:
https://github.com/betaflight/betaflight-configurator/blob/309bd431289df96c75092c163b15e378e441cae8/src/js/tabs/firmware_flasher.js#L579

Reply:
Added to autoDetect